apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -43,36 +43,223 @@ export class EmailOutputRendererUsecase {
     return { subject: subject as string, body: renderedHtml };
   }
 
-  private async parseTipTapNodeByLiquid(
-    tiptapNode: TipTapNode,
-    renderCommand: EmailOutputRendererCommand
-  ): Promise<TipTapNode> {
-    const parsedString = await parseLiquid(JSON.stringify(tiptapNode), renderCommand.fullPayloadForRender);
+  private async parseMailyContentByLiquid(
+    mailyContent: MailyJSONContent,
+    variables: FullPayloadForRender
+  ): Promise<MailyJSONContent> {
+    const parsedString = await this.parseLiquid(JSON.stringify(mailyContent), variables);
 
     return JSON.parse(parsedString);
   }
-}
 
-export const parseLiquid = async (value: string, variables: FullPayloadForRender): Promise<string> => {
-  const client = new Liquid({
-    outputEscape: (output) => {
-      return stringifyDataStructureWithSingleQuotes(output);
-    },
-  });
+  private async transformMailyContent(
```

Comment:
IMO we can move this to a separate function outside of this Usecase Class and create a more specific unit test, however, we can do so in separate PR.

Reply:
I'm not sure if `transformMailyContent` would ever be used outside of this usecase, I think for the clarity is better to keep all the logic contained in a single place. That said, if the need for separation emerges we can easily do that.